python/lancedb/remote/client.py

```diff
@@ -152,16 +152,11 @@ async def post(
 
     @_check_not_closed
     async def list_tables(
-        self, limit: int, page_token: Optional[str] = None
+        self, limit: int, page_token: Optional[str] = ""
```

Comment:
Does `page_token=None` break anything? We should prob fix where it breaks.

`Optional[blabla]` does expect the default value as `None`.